docs/_docs/monitoring-metrics/system-views.adoc

```diff
@@ -168,6 +168,35 @@ This view exposes information about currently running compute tasks started by a
 |USER_VERSION | string |  Task user version
 |===
 
+== JOBS
+
+This view exposes information about currently running compute job started on a node.
```

Comment:
"This system view shows a list of compute jobs started by a node as part of a compute task. To view the status of the compute task refer to the TASKS system view." - how about rephrasing this way?

Reply:
Done. Thanks.